mi_analysis.m

```diff
@@ -119,14 +133,107 @@ function buildMIs(obj, mi_data)
                 
                 if v > 2; disp([newline '--> Group ' num2str(iGroup) ' has ' num2str(max(size(x))) ' data points']); end
                 
-	            obj.arrMIcore(iGroup,:) = {core1 coeffs{iGroup,1} 0 key};
+	            obj.arrMIcore(iGroup,:) = {core1 coeffs{iGroup,1} NaN NaN NaN key};
                 
                 if v > 2; disp([newline '--> arrMIcore assigned']); end
-	            % BC: The obj.findMIs function basically calls run_sims
+	            % BC: The obj.calcMIs function basically calls run_sims
             end
             
-            if v > 4; mi_ksg_viz.audit_plots(obj); end
```

Comment:
I think we also want to keep this line. This is important. I'm wondering if we have different code versions floating around....